requirements.txt

```diff
@@ -12,6 +12,7 @@ pyfakefs==3.7.1
 pytest==5.3.5
 python-dateutil==2.8.1
 pytz==2019.3
+psutil==5.7.2
```

Comment:
I'm opposed to adding packages to requirements.txt because a fuzzer needs it. I don't think this is the right place.

Reply:
Should i add another requirements.txt inside base-image directory with just those 2-3 deps, i thought you preferred one requirements.txt.

Reply:
I prefer 1 requirements.txt for all of FuzzBench (not including fuzzers/benchmarks).
However, if a fuzzer or a benchmark needs Python packages, they need to install it in builder/runner.
Just like it would be wrong to add the apt package ninja to base-image because some random benchmark needs it, it is also wrong to add a python package because one benchmark needs it.

Reply:
psutil is a general utility, that is the reason i put it in requirements.txt [for other fuzzer use it is nice to print things with it]. currently people needs to add pylint hacks and move the import if some thing is installed in their runner docker. pick one option after this comment and i will either keep it as-is or move it back.

Reply:
> psutil is a general utility, that is the reason i put it in requirements.txt [for other fuzzer use it is nice to print things with it]. currently people needs to add pylint hacks and move the import if some thing is installed in their runner docker. pick one option after this comment and i will either keep it as-is or move it back.

I think it should not be installed anywhere general I think it should be installed only by the fuzzers that use it.
I thought the purpose of your change was to eliminate the second requirements.txt we had for the runner images right?

Reply:
yes that is the purpose, so reverting the psutil and keeping it to klee (as originally).